pkg/loop/config.go

```diff
@@ -150,3 +196,27 @@ func getDatabaseURL() (*url.URL, error) {
 	}
 	return u, nil
 }
+
+func getDatabaseIdleInTxSessionTimeout() (time.Duration, error) {
+	return time.ParseDuration(os.Getenv(envDatabaseDefaultIdleInTxSessionTimeout))
```

Comment:
do these need default values? if i'm reading the code correctly, i think this will error if the env vars aren't set? are we assuming that the core node sets them? if so, i think we need a comment somewhere in this repo, probably where they are defined, and a test integration test to ensure all the value are coming through the env correctly

Reply:
They are only required if the DB URL is included. Yeah the dummy plugin on the core side is meant to address it.